### PR TITLE
[#594] Fix system account initialization

### LINKED ISSUE #594: [BUG] Fix system account initialization
**Describe the bug**
The current database transaction used to create documents required to set up the initial system account does not set the `ServicesIDs` field in the created application organization, so permission validation fails when attempting to create the two initial permissions. Also, auth types and service registrations are not cached correctly on service initialization.

**To Reproduce**
1. Run the service with no data in the database
2. Observe error when caching zero auth types
3. Observe trying to assign `all_system_core` permission fails
4. Observe initial service registrations are not cached before attempting to load them from the cache (when not running on replica set)

**Expected behavior**
Service initialization should not throw errors unless validation on environment variables fails, or Mongo fails to insert or update documents.

### PR DESCRIPTION
**Pull request by roberlander2**
## Description
This PR fixes initial system account setup and cache initialization bugs.

**Resolves #594**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Review by shurwit — APPROVED** [review]
Thanks @roberlander2, LGTM! 

@petyos, please take a quick look when you get a chance. Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add default assigners permission [#477](https://github.com/rokwire/core-building-block/issues/477)
 
 ### Fixed
+- Fix system account initialization [#594](https://github.com/rokwire/core-building-block/issues/594)
```
Comment (petyos) [inline CHANGELOG.md:53]: It should be in the `Fixed` section but under `[Unreleased]`

**Review by petyos — DISMISSED** [review]
LGTM, one small comment in the CHANGELOG.md file

File: CHANGELOG.md (hunk 1 above)
Comment (shurwit) [reply CHANGELOG.md:53]: Thanks for the catch @petyos!

**Pull request merged**